liionsden/settings/settings.py

```diff
@@ -157,7 +157,10 @@
         },
         "console": {"class": "logging.StreamHandler", "formatter": "verbose"},
     },
-    "loggers": {"django": {"handlers": ["file", "console"], "level": "INFO"}},
+    "loggers": {
+        "django": {"handlers": ["file", "console"], "level": "INFO"},
+        "azure": {"handlers": ["file", "console"], "level": "DEBUG"},
```

Comment:
We should probably include this as a separate PR, as it doesn't directly relate to this issue. Could you undo the changes in this file and open a new PR just with this change separately please?